### ISSUE TITLE
Additional border for the list section

### ISSUE THREAD

**Issue by junegunn** [junegunn/fzf#4148]
# Summary

* Add `--list-border` option to draw additional border around the list section
* Also add `--list-label` and `--list-label-pos`, similar to `--border-label` and `--border-label-pos`, for placing label on the inner border

# Objective

* Provide greater flexibility for UI customization

# Examples

```sh
git ls-files | fzf \
    --preview 'bat --color=always {} --style=numbers' --info inline-right \
    --list-border --list-label ' Result ' --preview-label ' Preview '
```

<img width="855" alt="image" src="https://github.com/user-attachments/assets/e507c30d-5462-4a36-ba72-3cdba66b40bb" />

```sh
git ls-files | fzf \
    --preview 'bat --color=always {} --style=numbers' --info inline-right \
    --list-border \
    --bind 'result:transform-list-label:
        if [[ -z $FZF_QUERY ]]; then
          echo " $FZF_MATCH_COUNT items "
        else
          echo " $FZF_MATCH_COUNT matches for [$FZF_QUERY] "
        fi
        ' \
    --bind 'focus:transform-preview-label:[[ -n {} ]] && printf " Previewing [%s] " {}' \
    --bind 'ctrl-r:change-list-label( Reloading the list )+reload(sleep 2; git ls-files)' \
    --color 'list-border:#669966,list-label:#99cc99,preview-border:#9999cc,preview-label:#ccccff' \
    --border --border-label ' Demo ' --padding 1,2
```

<img width="855" alt="image" src="https://github.com/user-attachments/assets/8a740376-b33b-4ccd-8910-27f34b3de9db" />

```sh
git ls-files | fzf \
    --preview 'bat --color=always {} --style=numbers' --info inline-right \
    --list-border thinblock --preview-window border-thinblock \
    --list-label-pos bottom --preview-label-pos bottom \
    --bind 'result:transform-list-label:
        if [[ -z $FZF_QUERY ]]; then
          echo " $FZF_MATCH_COUNT items "
        else
          echo " $FZF_MATCH_COUNT matches for [$FZF_QUERY] "
        fi
        ' \
    --bind 'focus:transform-preview-label:[[ -n {} ]] && printf " Previewing [%s] " {}' \
    --bind 'ctrl-r:change-list-label( Reloading the list )+reload(sleep 2; git ls-files)' \
    --color 'bg:#222222,border:#333333,list-bg:#334433,preview-bg:#333344,list-border:#669966,list-label:#99cc99,preview-border:#9999cc,preview-label:#ccccff' \
    --border thinblock --border-label ' Demo ' --border-label-pos bottom --padding 1,2
```

<img width="855" alt="image" src="https://github.com/user-attachments/assets/72788dbf-337c-46ab-bf9c-5b6a351565f6" />

**Comment by bitraid** [comment]
Thank you for all for all your work and continuous enhancements.
It seems to work fine, you only forgot to add:  
```diff
diff --git a/src/options.go b/src/options.go
index bc0ddeb..b467b24 100644
--- a/src/options.go
+++ b/src/options.go
@@ -2637,6 +2637,10 @@ func parseOptions(index *int, opts *Options, allArgs []string) error {
 				if opts.ListBorderShape, err = parseBorder(value, false); err != nil {
 					return err
 				}
+			} else if match, value := optString(arg, "--list-border="); match {
+				if opts.BorderShape, err = parseBorder(value, false); err != nil {
+					return err
+				}
 			} else if match, value := optString(arg, "--list-label="); match {
 				opts.ListLabel.label = value
 			} else if match, value := optString(arg, "--list-label-pos="); match {
```

**Comment by junegunn** [comment]
@bitraid Thanks! Fixed in fd513f8.
